web/data/data.go

```diff
@@ -203,6 +214,11 @@ func UpdateDoc(c echo.Context) error {
 		}
 	}
 
+	errHoook := hooks.Run(instance, hooks.EventUpdate, doc, old)
```

Comment:
I'd have call this after the real update to avoid calling it if there the update fails (bad revision for example)